Mapping/AutoRouteDefinition.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Component\RoutingAuto\Mapping;
+
+/**
+ * Represents the definition of an auto route.
+ */
+class AutoRouteDefinition
+{
+    /**
+     * This constant will be used as the auto route definition's
+     * "name" when the deprecated string value for the "uri-schema" is used.
+     */
+    const DEPRECATED_NAME = '_default_';
+
+    /**
+     * @var string
+     */
+    private $uriSchema;
+
+    /**
+     * @var array
+     */
+    private $defaults = array();
+
+    public function __construct($uriSchema, array $defaults = array())
+    {
+        $this->uriSchema = $uriSchema;
+        $this->defaults = $defaults;
+    }
+
+    /**
+     * Return the URI schema.
+     *
+     * @return string
+     */
+    public function getUriSchema()
```

Comment:
This variable is represented by the key `template` in the metadata files, and the parent key of the list of definitions is labelled `uri_schema`. We should either:

- Rename this method to `template`
- Rename the parent key in the metadata to `definitions` and rename `template` to `uri_schema`.